### ISSUE TITLE
Deprecation warning with Rails 6

### ISSUE THREAD

**Issue by packerbacker89** [urvin-compliance/caracal-rails#8]
Using caracal-rails with Rails 6 leads to the following deprecation warning:

    DEPRECATION WARNING: Single arity template handlers are deprecated. Template handlers must
    now accept two parameters, the view object and the source for the view object.
    Change:
      >> Caracal::Rails::TemplateHandler.call(template)
    To:
      >> Caracal::Rails::TemplateHandler.call(template, source)

**Comment by dwood-somniainc** [comment]
Yup. I get this error in my testing suite all of the time. Hopefully, he'll be able to update it soon.

**Comment by spovich** [comment]
This is a one-liner fix in this PR: https://github.com/urvin-compliance/caracal-rails/pull/12

Verified this fixes the issue.  Any chance of a new release?
